project1.css

```diff
@@ -1,30 +1,33 @@
 .navbar {
-	background-color: #7a7676;
+	background-color: #fc7b03;
 	display: flex;
 	justify-content: center;
+	width: 100%;
+}
+h2 {
+	font-size: 2rem;
 }
-
 .nav-button {
     display: inline-block;
     border: none;
     padding: 1rem 2rem;
     margin: 0;
     text-decoration: none;
-    background: #7a7676;
-    color: #ffffff;
+    background: #fc7b03;
+    color: #000000;
     font-family: Arial;
-    font-size: 1rem;
+    font-size: 1.5rem;
     cursor: pointer;
     text-align: center;
```

Comment:
i think this can be removed too - they aren't very wide so they are just centered naturally